keyvault/data-plane/pom.xml

```diff
@@ -34,7 +34,7 @@
   </scm>
 
   <properties>
-    <azure-keyvault.version>1.2.0</azure-keyvault.version>
```

Comment:
Any reason you aren't using this version in all the individual keyvault pom files that is importing this as a parent pom? That would reduce copy and paste effort.

Reply:
After reading a little about using this approach it sounds like there are some weird issues so for now lets just make sure we update all the places.

Reply:
This is where having the versions.txt approach I discussed with @weshaggard would be very useful.

Reply:
I agree.